query-connector/src/app/queryBuilding/buildFromTemplates/page.tsx

```diff
@@ -88,7 +106,8 @@ export default function QueryTemplateSelection() {
             <Button
               className="margin-0"
               type={"button"}
-              disabled={!noTemplateSelected}
+              disabled={!atLeastOneItemSelected || !queryName}
```

Comment:
in the figma, it looks like the button is completely hidden before a selection was made. I think the disabled state better fits what we have on other pages, but i also know you raised on prior tickets that the disabled state is not generally preferred to just providing an error message indicating what they need to include. Not sure it needs to be fixed here though 🤷🏽

Reply:
ah, good catch! I will rework this per Michelle's revisions (from this [Slack thread](https://skylight-hq.slack.com/archives/C07PQ078KK2/p1731077429943419?thread_ts=1730997768.181319&cid=C07PQ078KK2)) , so that the button is hidden in an empty state, but shown and clickable when conditions are selected (which would trigger an error message if there is no Query Name input yet).